examples/package.json

```
@@ -19,6 +19,6 @@
   "scripts": {
     "lint": "eslint . --cache --ext .ts,.tsx --max-warnings 0",
     "lint-fix": "yarn lint --fix; exit 0",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit && echo 'Typehchecked successfully'"
```

```suggestion
    "typecheck": "tsc --noEmit && echo 'Typechecked successfully'"
```
Why do you need to echo the statement? :thinking:

---

Just to indicate that something is happening, we can remove it if you like, I don't feel strongly on i

---

No I guess it okay, as long as it doesn't make it less usable in the CI (but we'll look at that when we get there)

---

I'll remove it if it's a problem then :+1: